data/the emerald sword.txt

```diff
@@ -0,0 +1,796 @@
+			`	"A couple months, probably. It depends on how alien this thing is, and if it's encrypted or not, but I should at least figure out if there is any recoverable data in here."`
+			`	John says he'll drop you a message once his results come out, and you part ways and return to your ship.`
+
+mission "The Box 3"
+	landing
+	name "The box"
+	description "Discover the whereabouts of the box from an information broker in the pirate world of Stormhold."
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Independent" "Neutral"
+		near "Sol" 100
+		attributes spaceport
+	destination "Stormhold"
+	
+	to offer
+		has "event: cube stolen"
+	
+	on offer
+		conversation
+			`You receive a call from John, who has been working on the mysterious Sheragi black box for about a month now.`
+			`	"Captain <last>! I have some bad news and some worse news. Which one do you want to hear first?"`
+			choice
+				`	"The bad news."`
+					goto bad
+				`	"The worse news."`
+					goto worse
+			label bad
+			`	"The bad news is that this is a lot more complicated than I thought. I found out that the data in the box has survived, and can still be recovered after all this time, but it will need some specialized hardware and knowledge that we don't have yet."`
+			`	"And the worse news?" You ask.`
+			`	Well... Some people invaded my lab during the night, and stole a lot of the things I was working on. Thankfully my insurance covers most of it, and I still have most of my heavy expensive equipment, but the box... is gone.`
+				goto whatnow
+			label worse
+			`	"Well... Some people invaded my lab during the night, and stole a lot of the things I was working on. Thankfully my insurance covers most of it, and I still have most of my heavy expensive equipment, but the box... is gone.`
+			`	"And the bad news?"`
+			`	"The bad news is that this is a lot more complicated than I thought. I found out that the data in the box has survived, and can still be recovered after all this time, but it will need some specialized hardware and knowledge that we don't have yet."`
+			label whatnow
+			`	"What do we do now? Do you have any idea where to find the box? Did you contact law enforcement?"`
+			`	"I did what I could, but if this was mandated as I expect, the police isn't going to help, and the box is probably off-world by now. However... if you have some cash at hand, and you're willing to go the extra mile to recover the box, there is a way."`
+			choice
+				`	"What do you mean?"`
+					goto info
+				`	"Are you trying to extort me?"`
+			`	"What? No, of course not! Do you think I would steal my own work? Let me explain...`
+			label info
+			`	"Albert's methods are... nonstandard, to say the least, and we've dealt with a certain individual once. An information dealer, who lives in Stormhold. Sketchy person, seems to have informants nearly everywhere, and knows about everything. They're known as Mammon. If you want to find the box, they'll probably know where it is."`
```

Comment:
```suggestion
			`	"Albert's methods are... nonstandard, to say the least, and we've dealt with a certain individual once. An information dealer who lives in Stormhold. Sketchy person, seems to have informants nearly everywhere and knows about everything. The individual in question is known as Mammon. If you want to find the box, dealing with Mammon is probably your best bet."`
```
I understand that the use of the word is to prevent the implication of any genders, but it felt somewhat awkward to me so I felt this was a better way to phrase it. Feel free to change that up if you don't like it.